README.md

```diff
@@ -29,6 +29,15 @@ Resolution supports different decentralized domains. Please, refer to the
 For more information, see our detailed
 [API Reference](https://unstoppabledomains.github.io/resolution/).
 
+### ENS support
+**Note: Ethereum Name Service requires installing additional packages 
+otherwise library throws error when trying to resolve ENS domain.**
+
+#### Required packages
```

Comment:
this is packages that will come with resolution lib when they install it right? we don't need to include this section